settings.gradle

```diff
@@ -142,6 +142,8 @@ include(':envoy')
 
 include(':grpc-proxy')
 
+include (':graphviz')
```

Comment:
Moved into qst. Our javadocs are currently inclusive, not exclusive - so I don't think there is currently a way to "exclude"... it will be excluded by default.